src/context/Trash-Content.js

```diff
@@ -0,0 +1,21 @@
+import React,{createContext,useState,useContext} from "react"
+
+const TrashContext = createContext()
+
+const TrashProvider = ({children}) => {
+    const [trashNotes, setTrashNote] = useState([])
+
+    const moveToTrash = (trashNote) =>  setTrashNote(prev => [...prev,trashNote])
+
+    const deleteFromTrash = (trashNote) => setTrashNote(prev => prev.filter(Obj => Obj.id !== trashNote.id))
+
```

Comment:
You can use proper variable naming like instead of Obj you can use camelCase  naming like obj or trashNoteObj, etc.